### PR TITLE
SAK-41210 Instructor feedback: Suggest previous inputs

### PR DESCRIPTION
**Pull request by mpellicer**

### CODE REVIEW

**Comment by mpellicer** [comment]
> Need to bind the data to a specific var for shared devices.

Addressed adding the userId.

File: library/src/webapp/js/sakai-reminder.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    SakaiReminder.prototype = {
+        init: function() {
+        },
+
+        getAll: function() {
+            return JSON.parse(localStorage.getItem(portal.user.id + '-sakaiReminder'));
+        },
+
+        new: function(text) {
+            var predictions = JSON.parse(localStorage.getItem(portal.user.id + '-sakaiReminder'));
```
Comment (fostersdesign) [inline library/src/webapp/js/sakai-reminder.js]: What does portal.user.id map to?

Comment (mpellicer) [reply library/src/webapp/js/sakai-reminder.js]: The user internal id, what we agreed on the core call meeting.

**Comment by mpellicer** [comment]
Should we wait to the T&L comments? I'm asking because will be good if we block this PR for merging.

File: library/src/webapp/js/sakai-reminder.js (hunk 1 above)
Comment (fostersdesign) [reply library/src/webapp/js/sakai-reminder.js]: Thanks for confirming, @mpellicer!

**Comment by fostersdesign** [comment]
Yes, we decided to wait for community groups' input before merging. Thanks.

**Comment by mpellicer** [comment]
> Yes, we decided to wait for community groups' input before merging. Thanks.

Thanks @fostersdesign! I've set the WIP label.

**Pull request merged**